### PR TITLE
Updated unique terms to work with previous functionality, and fixed a  pagination bug

### PR DESCRIPTION
**Pull request by dionboles-asym**
This branch I updated unique terms to reference previous functionality and I fixed a pagination bug. I also had to remove the run method from all of my test.

### CODE REVIEW

File: cdapython/results/page_result.py [hunk 1]
```diff
@@ -257,8 +257,13 @@ def next_page(
...
                 show_term_count=show_term_count,
             )
-            self.total_row_count = next_result.total_row_count
-            return next_result
+
+            if next_result.total_row_count is not None:
+                self.total_row_count = next_result.total_row_count
+                return next_result
+            return None
+            # self.total_row_count = next_result.total_row_count
```
Comment (otchet-broad) [inline cdapython/results/page_result.py:265]: Please remove commented out code

File: tests/test_no_embedded_pandas.py
Comment (otchet-broad) [inline tests/test_no_embedded_pandas.py:1]: The name of this file implies there's no embedded pandas - but that's not the case.
Could you please change the name of the file to something like:
test_demonstrate_using_pandas_outside_of_cda_python.py

**Review by otchet-broad — CHANGES_REQUESTED** [review]
Thanks for the PR.

I left a few comments for some things that need cleanup, but overall seems good.  I'm glad you're seeing improvements and identifying issues by adding test coverage.

File: cdapython/utils/utility.py [hunk 2]
```diff
@@ -101,23 +101,51 @@ def get_table_version() -> str:
...
+    verify: Optional[bool] = None,
+    async_req: Optional[bool] = True,
+    show_sql: bool = False,
+    show_counts: bool = False,
+    show_term_count: bool = False,
+    verbose: bool = True,
+    limit: int = 100,
+) -> Paged_Result:
     """
     Show all unique terms for a given column.
```
Comment (otchet-broad) [inline cdapython/utils/utility.py:115]: The _description_ is empty on each of these.  Can you please explain what each are supposed to do?

File: cdapython/utils/utility.py [hunk 3]
```diff
@@ -111,20 +111,21 @@ def unique_terms(
...
-        verbose (bool, optional): _description_. Defaults to True.
-        limit (int, optional): _description_. Defaults to 100.
+        col_name (str): This is the default way to search for a unique term from the CDA service API.
+        system (str, optional): his is an optional parameter used to filter the search values by data center, such as "GDC", "IDC", "PDC", or "CDS". Defaults to "".
+        offset (int, optional): The number of entries to skip. Defaults to 0.
+        host (Optional[str], optional): This is where the user can set a host for a different server. Defaults to None.
+        verify (Optional[bool], optional): This will send a request to the cda server without verifying the SSL Cert Verification. Defaults to None.
+        async_req (Optional[bool], optional): Execute request asynchronously. Defaults to True.
+        show_sql (bool, optional): This will show the sql returned from the server. Defaults to False.
+        show_counts (bool, optional): Deprecated will be removed: This will show or hide the count. Defaults to False.
```
Comment (otchet-broad) [inline cdapython/utils/utility.py:125]: Is this really deprecated or nonfunctional?

Comment (otchet-broad) [reply cdapython/utils/utility.py:125]: If it's nonfunctional in the current state of the code and needs to be functional - state that.  If it's functional and going away, deprecated is the correct term.

File: cdapython/utils/utility.py [hunk 4]
```diff
@@ -17,7 +17,7 @@
 from cdapython.exceptions.custom_exception import HTTP_ERROR_API, HTTP_ERROR_SERVICE
 from cdapython.results.columns_result import ColumnsResult
 from cdapython.results.factories.collect_result import CollectResult
-from cdapython.results.page_result import get_query_result
+from cdapython.results.page_result import Paged_Result, get_query_result
```
Comment (ahaessly) [inline cdapython/utils/utility.py:20]: a nit pick - usually I would expect to see either PagedResult or paged_result. This seems like a strange mix of the two. was there a specific reason to do this?

File: tests/test_C_integration.py [hunk 5]
```diff
@@ -10,7 +10,7 @@ def test_basic_integration() -> None:
 
 
 def test_unique_terms() -> None:
-    terms = unique_terms("sex", "GDC").run(host=host, show_term_count=True)
+    terms = unique_terms(col_name="sex", system="GDC", host=host, show_term_count=True)
```
Comment (ahaessly) [inline tests/test_C_integration.py:13]: does this require a change to the documentation? If so, I think we should add a ticket for Amanda so she knows what to update in the documentation.

**Review by ahaessly — APPROVED** [review]
a couple of minor comments

File: tests/test_C_integration.py (hunk 5 above)
Comment (otchet-broad) [reply tests/test_C_integration.py:13]: This brings the functionality back in line with the documentation.

File: cdapython/utils/utility.py (hunk 3 above)
Comment (dionboles-asym) [reply cdapython/utils/utility.py:125]: The show_counts param currently nonfunctional. it is currently waiting on a response to see if it is needed to re-implement.

**Pull request merged**